lib/data_update_scripts/20230518035338_add_hero_billboard_feature_flag.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class AddHeroBillboardFeatureFlag
+    def run
```

Comment:
I don't really think that we need to add this Data Update Script for the feature flag especially if its going to be mostly tested on DEV for now, it feels like it can be just manually added. Adding it as a Data Update Script means that when we want to remove the feature flag it will include a code change too, but I don't feel too strongly about it and we can go with either.

Reply:
When you remove and deploy this you can just tell the community team to **add and enable** the `hero_billboard` feature flag through /admin/feature_flags/features.